src/app/validators/User.js

```diff
@@ -15,7 +15,7 @@ class UserValidator {
     });
 
     // Validating the input data
```

Comment:
Can you remove that comment too? I don't remember why I wrote that, it's kind of unnecessary if you know the meaning/semantic of the function  🤦‍♀️

Reply:
done 😉